### ISSUE TITLE
How to assign ground truth for training ?

### ISSUE THREAD

**Issue by GabbySuwichaya** [almazan/deep-image-retrieval#24]
Hello, Thank you for releasing the scripts and sharing the awesome work. 
Here, I would like to ask about the training label. I have a look at the Multistaged backpropagation algorithm in [Supp](https://openaccess.thecvf.com/content_ICCV_2019/supplemental/Revaud_Learning_With_Average_ICCV_2019_supplemental.pdf). 

I wonder how did you assign the binary-ground truth for training? I know each of the entries in binary-ground truth indicates the similarity of the images in a batch. 
But, since there are a number of training images from [Landmarks clean](http://sites.skoltech.ru/compvision/projects/neuralcodes/), so I wonder how did you generate the binary-ground truth. 

Your information will greatly help me verify my implementation with yours.

**Comment by almazan** [comment]
Hi @GabbySuwichaya,

This binary label indicates whether an image belongs to the same class as the query. This means that a given image can have a different label depending the query it is compared to.

I hope this helped, but let me know if you have further questions.
